pkg/vsphere/tasks/waiter.go

```diff
@@ -108,10 +108,10 @@ func isRetryError(op trace.Operation, err error) bool {
 		switch f := soap.ToSoapFault(err).VimFault().(type) {
 		case types.TaskInProgress:
 			return true
-		case *types.NetworkDisruptedAndConfigRolledBack:
+		case types.NetworkDisruptedAndConfigRolledBack:
```

Comment:
This is a slight fix for an issue that has not been filed against checks here not working previously. We may not have seen it yet. But we will fix it here and move on.